site/content/en/docs/manual/advanced/serverless-tutorial.md

```diff
@@ -161,7 +160,6 @@ Finally you will get bounding boxes.
 ![SiamMask results](/images/siammask_results.gif)
 
 `SiamMask` model is more optimized to work on Nvidia GPUs.
-For more information about deploying the model for the GPU, [read on](#objects-segmentation-using-mask-rcnn).
```

Comment:
Lets not remove documentation about GPU functions deployment. As a bit outdated documentation is better than no docs at all.